### DISCUSSION TITLE
A4a Form

### DISCUSSION THREAD

**Discussion by SuzanneHunt-KC** [naccdata/uniform-data-set#69] [Forms]
A4a #1 The pdf says to end the form if the participant has not enrolled in a clinical trial.  Wouldn't you want them to answer question #3 (amyloid-related imaging abnormalities)?

Also, I'm not sure that participants know what is meant by "modify ADRD biomarkers".

**Comment by SuzanneHunt-KC** [comment]
We discussed some of the mechanics of actually filling out the form in the EDC meeting today.

@mllerch @mcleod42  I noticed in the newest A4a pdf in github, question #1 still indicates the form should end if the participant hasn't participated in a trial.  However, that would mean you wouldn't get an answer to #3.  Is that really what you intend?

**Comment by SuzanneHunt-KC** [comment]
1/5/2024

I am making notes here about changes I made to REDCap, or changes that I recommend.

I added a line the instructions providing a link to ClinicalTrials.

A link to  the coding guidebook should be added to the instructions.  the URL is unknown at this time.

Please address the 2 previous comments.  As it stands, question #3 will often get skipped.

**Comment by jculhane** [comment]
@SuzanneHunt-KC 

"A4a Q1 The pdf says to end the form if the participant has not enrolled in a clinical trial. Wouldn't you want them to answer Q3 (amyloid-related imaging abnormalities)?" 
Response: No, Q3 asks if the participant experienced ARIA or other major adverse events associated with **treatments expected to modify ADRD biomarkers**. It should not be asked if they haven't received these treatments.


"Also, I'm not sure that participants know what is meant by "modify ADRD biomarkers"."
Response: This is a clinician-facing form, so the clinician can explain further if needed.


"A link to the coding guidebook should be added to the instructions. the URL is unknown at this time."
Response: Correct, we will update/link later when the guidebooks are available on the NACC website.
